README.rst

```diff
@@ -32,9 +32,9 @@ See `doc/install.txt`_ in the source code archive for general information. Excep
 
 .. _doc/install.txt: doc/install.txt
 
-Python 2.7.2 or later is needed. It doesn't work with Python 3 yet, see `#40 <https://github.com/linkchecker/linkchecker/pull/40>`_ for details.
+Python 3.5.0 or later is needed.
 
-The version in the pip repository is old. Instead, you can use pip to install the latest release from git: ``pip install git+https://github.com/linkchecker/linkchecker.git@v9.4.0``. See `#4 <https://github.com/linkchecker/linkchecker/pull/4>`_.
+The version in the pip repository is old. Instead, you can use pip to install the latest release from git: ``pip install git+https://github.com/linkchecker/linkchecker.git@v10.0.0rc1``. See `#4 <https://github.com/linkchecker/linkchecker/pull/4>`_.
```

Comment:
Do we have a v10.0.0rc1 tag yet on github?  I don't see one on https://github.com/linkchecker/linkchecker/tags.

Maybe we should use `@master` for now.

Reply:
What about this?  Do we want non-working instructions on the front page of the GitHub repo?

Reply:
nope. i guess after merging this we'd need to immediately tag rc1. this is a downside of the github workflow, i guess ;)

Reply:
If we merge this at the same time we release 10.0.0rc1, there's little point in instructing people how to pip install from git, they can pip install 10.0.0rc1 from pypi just as well, if they use `pip install --pre linkchecker`.

Reply:
how about we instruct people to install from git HEAD there instead? something like:

```suggestion
The version in the pip repository is old. Instead, you can use pip to install the latest from git: ``pip install git+https://github.com/linkchecker/linkchecker.git``. See `#4 <https://github.com/linkchecker/linkchecker/pull/4>`_.
```

that way we don't have to repeatedly update that darn file...

Reply:
That's where I was going with my "Maybe we should use @master for now." suggestion earlier, but you're right, `@master` is redundant and not necessary.

Reply:
Let's get rid of the "pip repository is old" sentence. It is the current release now.

Reply:
Or it will be if we merge this PR right before releasing 10.0.0.

Reply:
We've already got `pip3 install LinkChecker` in install.txt...

That starts to make me think we don't want "install from git" the only instruction in the README. And that this should move as an alternative to install.txt.

Reply:
> That starts to make me think we don't want "install from git" the only instruction in the README. And that this should move as an alternative to install.txt.

Ignore me...